pax/config/Simulation.ini

```diff
@@ -5,3 +5,1571 @@ look_for_config_in_runs_db = False
 input = 'WaveformSimulator.WaveformSimulatorFromCSV'
 decoder_plugin = None
 pre_dsp = 'ZLE.SoftwareZLE'
+
+[WaveformSimulator]
+# an example of PMT after pulses configuration for each PMT
+# the precise one for Xe1T will come later after analysis
+# this is just for making tester happy
+each_pmt_afterpulse_types = {
+    0:{
```

Comment:
@mcfatelin If these are all the same, you can just do a 'dictionary comprehension:      `{ i : { 'CH4+':  {'p': 2. * 1.8/4.9, 'time_distribution': 'normal', 'time_parameters': dict(loc=1 * us, scale=10. * ns)} for i in range(255)}`

Try this in a command line: `{i : i * 2 for i in range(10)}` to see what I mean